src/sgame/sg_main.cpp

```diff
@@ -2331,10 +2331,24 @@ void G_RunFrame( int levelTime )
 		return;
 	}
 
+	int matchTime = levelTime - level.startTime;
+
+	/* Don't process frame at matchTime 0 as delay will be zero and
```

Comment:
All the divisions by zero fixed in that PR were caught with 1ms frametime/1000fps limit.

The fixes for divisions by zero happening with infinite frametime will be implemented in another PR (I caught some and I still want to fix some of them because it's useful when benchmarking, as benchmarking a timer looks silly to me).